frontend/javascripts/viewer/view/distance_measurement_tooltip.tsx

```diff
@@ -101,29 +113,19 @@ export default function DistanceMeasurementTooltip() {
     );
   }
 
-  const {
-    left: viewportLeft,
-    top: viewportTop,
-    width: viewportWidth,
-    height: viewportHeight,
-  } = getInputCatcherRect(state, orthoView);
-  const tooltipPosition = calculateMaybePlaneScreenPos(state, position, orthoView);
-
-  if (tooltipPosition == null) {
-    return null;
-  }
-
```

Comment:
This is now obsolete due to storeing the measurement tool tip viewport pos directly in the store. No conversion via `calculateMaybePlaneScreenPos` needed anymore